curvefs/src/client/kvclient/kvclient_manager.cpp

```diff
@@ -64,8 +64,10 @@ void KVClientManager::Uninit() {
 void KVClientManager::Set(std::shared_ptr<SetKVCacheTask> task) {
     threadPool_.Enqueue([task, this]() {
         std::string error_log;
+        kvClientManagerMetric_->setQueueSize << 1;
```

Comment:
when enqueue, the setQueueSize should plus one. So the code `kvClientManagerMetric_->setQueueSize << 1`should be in front of `threadPool_.Enqueue([task, this]() {}`